tests/clients/test_device_settings.py

```diff
@@ -403,6 +403,15 @@ def device_settings_with_multiple_values():
     return device_settings_dict
 
 
+@pytest.fixture
+def device_settings_with_locked_backup_set():
+    device_settings_dict = deepcopy(DEVICE_DICT_W_SETTINGS)
+    device_settings_dict["settings"]["serviceBackupConfig"]["backupConfig"][
+        "backupSets"
```

Comment:
Shouldn't this be changing the whole structure to be a `dict` as well rather than a list?

Reply:
No, because here we're only locking a single backup set. The change from list > dict happens when you lock the _number_ of backup sets, preventing a user from creating any new sets.